### ISSUE TITLE
View / edit basic user and organization settings from settings page

### ISSUE THREAD

**Issue by CloudNiner** [azavea/temperate#533]
## Overview

Users need a place to be able to update basic information about themselves and their organization. This view is different from the city profile page, which is a place for users to provide information specific to filling out their adaptation plan.

#### Link to settings page in navbar user dropdown

![screen shot 2018-01-31 at 9 23 59 am](https://user-images.githubusercontent.com/1818302/35627963-8093555c-0668-11e8-9b98-5da8f031aba1.png)

#### User/Org profile page

![screen shot 2018-01-31 at 9 24 51 am](https://user-images.githubusercontent.com/1818302/35628010-a1aeac96-0668-11e8-87a6-185d8a2c94e2.png)

#### User editing a field

![screen shot 2018-01-31 at 9 25 17 am](https://user-images.githubusercontent.com/1818302/35628033-b1ee1ac4-0668-11e8-9b91-b76a437d9e9d.png)

#### Toast when user updates organization name

![screen shot 2018-01-31 at 9 26 13 am](https://user-images.githubusercontent.com/1818302/35628076-cd4e246c-0668-11e8-8c33-779a4044cef3.png)

## Developer tasks

- Consider implementing the basic page routing as an initial PR, to unblock further work on the "Invite Colleagues" control, which is a separate task

## Done criteria

- [x] User can navigate to separate usr/org settings page
- [ ] User can edit fields by clicking the edit icon, then making a change, then clicking the save/cancel buttons to update/discard the change respectively
- [ ] User can edit the following fields using the workflow in the previous item:
  - First name
  - Last name
  - Organization name
- [ ] User can view but not edit the following fields
  - email
  - Organization city
- [x] User can use the "Back to {{ view }}" link at the top of the page to return to the page they were on before navigating to this page. We should not always return them to the dashboard.
